x-pack/plugins/monitoring/server/lib/cluster/get_clusters_from_request.ts

```diff
@@ -168,17 +169,40 @@ export async function getClustersFromRequest(
     }
   }
   // add kibana data
-  const kibanas =
+  const [kibanas, kibanaClusterRules, kibanaInstanceRules] =
     isInCodePath(codePaths, [CODE_PATH_KIBANA]) && !isStandaloneCluster
-      ? await getKibanasForClusters(req, clusters, CCS_REMOTE_PATTERN)
-      : [];
+      ? await Promise.all([
+          getKibanasForClusters(req, clusters, CCS_REMOTE_PATTERN),
+          getClusterRuleDataForClusters(req, clusters, CCS_REMOTE_PATTERN),
+          getInstanceRuleDataForClusters(req, clusters, CCS_REMOTE_PATTERN),
+        ])
+      : [[], []];
   // add the kibana data to each cluster
   kibanas.forEach((kibana) => {
     const clusterIndex = clusters.findIndex(
       (cluster) =>
         get(cluster, 'elasticsearch.cluster.id', cluster.cluster_uuid) === kibana.clusterUuid
     );
     set(clusters[clusterIndex], 'kibana', kibana.stats);
+
+    const clusterKibanaRules =
+      kibanaClusterRules === null
```

Comment:
I don't think this condition could ever be met because `getClusterRuleDataForClusters` returns an array of clusters' data that could have `null` values instead of just being `null`